engine/liveupdate/src/job_thread.cpp

```diff
@@ -169,9 +169,6 @@ void Update(HContext context)
 {
     DM_PROFILE("Update");
 
-    if (context->m_ThreadContext.m_Done.Empty())
-        return;
-
```

Comment:
It was an early out, but it wasn't protected with the mutex.

Also, I think it was a bug, since in HTML5 (which doesn't use the threads), the early out was done _before_ any work wark ever done (UpdateSingleThread on the line before)